### ISSUE TITLE
Chapter 10 MVP List

### ISSUE THREAD

**Issue by satsie** [saving-satoshi/saving-satoshi#1192]
This is the prioritized list of bugs and features that need to be done before we can release chapter 10. The timeline is as follows:

**Saturday November 30**: be code complete
**Sunday December 1**: cut the release (v 1.0?!?!?!??!) and deploy to prod
**Monday December 2**: official announcement

Leave a comment if you disagree with anything here or feel something is missing. For anything not on this list the plan is to do follow up releases.

- [x] https://github.com/saving-satoshi/saving-satoshi/issues/1183
- [x] https://github.com/saving-satoshi/saving-satoshi/issues/1175 (I think this is done but I need to double check)
- [x] https://github.com/saving-satoshi/saving-satoshi/issues/1189
- [x] https://github.com/saving-satoshi/saving-satoshi/issues/1188
- [x] https://github.com/saving-satoshi/saving-satoshi/issues/1190
- [x] https://github.com/saving-satoshi/saving-satoshi-script/issues/13

Stretch goal:
- [x] https://github.com/saving-satoshi/saving-satoshi/issues/1138
- [x] https://github.com/saving-satoshi/saving-satoshi/issues/1191

**Comment by benalleng** [comment]
Closed as of #1239
